aviatrix-aws-existing-controller-ha.json

```diff
@@ -23,7 +23,9 @@
          "AviatrixTagParam": { "default" : "Enter Name tag of the existing Aviatrix Controller instance." },
          "S3BucketBackupParam": { "default" : "Enter S3 Bucket which will be used to store backup files." },
          "NotifEmailParam": { "default" : "Enter an email to receive notifications for autoscaling group events" },
-         "PrivateAccess": { "default" : "Enter True to enable Private IP Access from lambda to the Controller. Please note that you have to attach the Lambda to the VPC subnet and ensure lambda has internet access via EIP/NAT" }
+         "PrivateAccess": { "default" : "Enter True to enable Private IP Access from lambda to the Controller. Please note that you have to attach the Lambda to the VPC subnet and ensure lambda has internet access via EIP/NAT" },
+         "AviatrixAppRoleName": {"default" : "Enter the controller IAM APP role name."},
+         "AviatrixEC2RoleName": {"default" : "Enter the controller IAM EC2 role name."}
```

Comment:
Could you change to CustomAviatrixAppRoleName and CustomAviatrixAppRoleName ?
Same for the environment variabless